services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/base_scheduler.py

```diff
@@ -202,6 +212,10 @@ async def _start_tasks(
     async def _stop_tasks(self, tasks: List[CompTaskAtDB]) -> None:
         ...
 
+    @abstractmethod
+    async def _reconnect_backend(self) -> None:
+        ...
```

Comment:
how does this work exactly? I'm referring to the `...` operator

Reply:
this behaves the same as pass. but since there is the decorator on top, if you try to instantiate this class it will fail